airflow/www/templates/airflow/dags.html

```diff
@@ -187,8 +212,23 @@
   <script src="{{ url_for('static', filename='d3.v3.min.js') }}"></script>
   <script src="{{ url_for('static', filename='jquery.dataTables.min.js') }}"></script>
   <script src="{{ url_for('static', filename='bootstrap-toggle.min.js') }}"></script>
+  <script src="{{ url_for('static', filename='bootstrap3-typeahead.min.js') }}"></script>
   <script>
 
+      $('#dag_query').on('keypress', function (e) {
+
+        if (e.which === 13) {
```

Comment:
what does this mean

Reply:
13 is enter, and I don't think there is any constant that can be used instead, at least possibly not portably.

Reply:
Right, adding a comment about that here.